### DISCUSSION TITLE
Better usage of `severity` issue tag

### DISCUSSION THREAD

**Discussion by ngxson** [ggml-org/llama.cpp#8097] [General]
When creating new issues, the [high severity](https://github.com/ggerganov/llama.cpp/issues?q=is%3Aissue+is%3Aopen+label%3A%22high+severity%22) and [critical severity](https://github.com/ggerganov/llama.cpp/issues?q=is%3Aissue+is%3Aopen+label%3A%22critical+severity%22) labels seem to be misused very often.

This is some of my ideas to improve the situation:
- Replace them with something more specific, for example: `inference quality`, `crash`, `gpu-related`
- Change issue template to:
  - `Issue related to llama-cli`
  - `Issue related to llama-server`
  - `Issue related to specific models`
  - `Issue related to core library`
  - `Other issues`

What do you think about this @mofosyne ?

**Comment by mofosyne** [comment]
Yeah on second thought, it may be a bit naive to let the public decide how urgent their issue is. I think we should still have the label available, but assigned by the team instead.

But it does make sense to split by subsystems. Happy to hear others thoughts, but if you can make a PR with your proposal you have my support at least.

**Reply by ngxson** [reply]
Thanks for the feedback. I'll think deeper on what's the best way to split the issue into appropriate categories. Will make a PR later when it's more clear for me.

I also agree to keep these labels, but only use by collaborators.
